docs/docs/modeling.md

```diff
@@ -65,7 +65,7 @@ classDiagram
 
     class TenantsWork {
         TenantId tenantId
-        AssignedCategory assignedCategory
+        List [AssignedCategory]
```

Comment:
Could you modify the multiplicity too because `TenantsWork` will now have more than 1 `AssignedCategory`.

Reply:
Thank you for your review. I revised this part. Could you please review and confirm?